src/kaleidoscope/MatrixAddr.h

```diff
@@ -90,6 +92,10 @@ class MatrixAddr {
     offset_ = this->row() * cols + c;
   }
 
+  void clear() {
```

Comment:
This is absolutely not criticism and is not something that I've been asking for to date, but I'd like us to get better about adding some level of inline doc for functions and methods as we add them.  What I'd be looking for is one or two lines of inline comment above the method explaining why you might call it (and in cases where the calling convention isn't blindingly obvious, how.)

If it's not part of your process, I don't think it makes sense to put a ton of effort into this until we're agreed that a PR's code looks ready to go or the code is harder to understand.

(This particular method is pretty obvious. ;)

Reply:
I totally agree. I think failing to do so this time was a bit uncharacteristic. I will happily add those comments once you make a decision on the function names.